packages/dx-react-chart/docs/guides/plugin-overview.md

```diff
@@ -2,14 +2,38 @@
 
 The Chart component uses plugin components to visualizes data specified via the `data` property. These components implement particular features and should be defined within the Chart component.
 
+## Plugin Types
+
 The React Chart supports the following plugin types:
 
-- **UI plugins**
- Use the provided data to render UI elements.
+- **Animation plugins**
```

Comment:
There should be two spaces at the end of line - the description part is rendered of the same line otherwise.

Reply:
I think it is better if the description part would be at the same line. 
Maybe it is more correct make description with lower case, like in the grid [plugin overview](https://devexpress.github.io/devextreme-reactive/react/grid/docs/guides/plugin-overview/)

How it looks now:
![1](https://user-images.githubusercontent.com/8238559/51245903-0d482280-199a-11e9-8a7d-e10a5e2ff1d1.png)

How it looks with description on another line:
![2](https://user-images.githubusercontent.com/8238559/51245927-1d600200-199a-11e9-8ae9-a1ecbbbcf063.png)

Reply:
Let it be like in grid then.